A/argp_standalone/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "argp_standalone"
+version = v"1.3.4"
```

Comment:
Shouldn't this be
```suggestion
version = v"1.3.0"
```
?